### PR TITLE
update docs for team admin permissions++

### LINKED ISSUE #5342: update docs for delegating responsibilities to a team admin
### Problem description

update docs for delegating responsibilities to a team admin

### Suggestions for a fix

- [x] update docs for delegating responsibilities to a team admin
- [x] review the console docs and make sure we are always using a modifier on the word admin, so either org admin, or team admin

### LINKED ISSUE #5367: console docs updates for changes to permissions for stack admins
### Problem description

we need docs for the work related to https://github.com/pulumi/pulumi-service/issues/6223

### PR DESCRIPTION
**Pull request by susanev**
### Proposed changes

this is updates for related team admin features. see https://github.com/pulumi/pulumi-service/issues/6210 for an overview of that work, or https://github.com/pulumi/pulumi-service/issues/6210 for the full epic definition.

i also removed "click" from the console docs in this pr.

@davidwrede i noticed we used a mix of `admin` and `administrator` in the console docs, would you mind if i standardized to `admin`, that's what we use in the console itself?

### Related issues (optional)
fixes #5367
fixes #5342

### CODE REVIEW

File: content/docs/intro/console/accounts-and-organizations/organizations.md
Comment (susanev) [inline content/docs/intro/console/accounts-and-organizations/organizations.md:149]:
```suggestion
| Settings | A place for you to manage your organization settings including subscription and payment information and history, stack permissions, and links to Pulumi's [continuous delivery guides]({{< relref "/docs/guides/continuous-delivery" >}}). |
```

File: content/docs/intro/console/collaboration/organization-roles.md
Comment (susanev) [inline content/docs/intro/console/collaboration/organization-roles.md:17]:
```suggestion
organization administrator or you must submit a request to the organization administrator for approval. In addition, depending on the [organization type]({{< relref "/docs/intro/console/accounts-and-organizations#organization-types" >}}), you must also be a member of the third-party organization or group backing the Pulumi
```

File: content/docs/intro/console/collaboration/organization-roles.md
Comment (susanev) [inline content/docs/intro/console/collaboration/organization-roles.md:63]:
```suggestion
Organization admins can toggler whether members have the ability to create teams.
```

File: content/docs/intro/console/collaboration/teams.md
Comment (susanev) [inline content/docs/intro/console/collaboration/teams.md:23]:
```suggestion
Members of a team can be granted `Team admin` or `Team member` permissions. Team admin's can add members to a
team. Both team admins and team members can grant stack access to a team. By default, any new team members will be
assigned the team member role. To change a team member's role use the 3-dot menu at the end of the table row.
```

File: content/docs/intro/console/collaboration/teams.md
Comment (susanev) [inline content/docs/intro/console/collaboration/teams.md:29]:
```suggestion
Organization admins can add a new team by going to the organization's **Teams** tab, and then
selecting **Create team**. Organization members can also be granted permissions to create teams
in the organization settings. Any member who creates a team will automatically be assigned the team admin role.
```

File: content/docs/intro/console/collaboration/project-and-stack-management.md
Comment (davidwrede) [inline content/docs/intro/console/collaboration/project-and-stack-management.md:37]:
```suggestion
The **Projects** tab displays a card-based view of your stacks with relevant details including the project name and description, language, stack name, last update information, and resource count. This example shows an organization backed by GitHub and sorts the stacks by their GitHub repositories. By default, stacks are grouped by [project]({{< relref "/docs/intro/concepts/project" >}}), but you may group them by [tag]({{< relref "/docs/intro/concepts/stack#stack-tags" >}}). Select the stack name to get more details on a specific stack.
```

File: content/docs/intro/console/collaboration/project-and-stack-management.md
Comment (davidwrede) [inline content/docs/intro/console/collaboration/project-and-stack-management.md:77]:
```suggestion
Select a resource link from **Activity > Timeline** to drill into a specific resource's properties and dependencies, if any. The **Resources** tab lets you toggle between a list and a graph view.
```

File: content/docs/intro/console/collaboration/stack-permissions.md
Comment (davidwrede) [inline content/docs/intro/console/collaboration/stack-permissions.md:44]:
```suggestion
- **Organization Settings**. An organization admin can configure [stack permissions for all members]({{< relref "organization-roles#organization-settings" >}}) for the organization's stacks, granting all members of the organization a minimum permission level.
```

File: content/docs/intro/console/collaboration/teams.md
Comment (davidwrede) [inline content/docs/intro/console/collaboration/teams.md:23]: Does three-dot mean ellipsis in this context?

```suggestion
assigned the team member role. To change a team member's role use the ellipsis menu item at the end of the table row.
```

File: content/docs/intro/console/collaboration/teams.md
Comment (davidwrede) [inline content/docs/intro/console/collaboration/teams.md:28]:
```suggestion
Organization admins can add a new team by going to the organization's **Teams** tab and 
selecting **Create team**. Organization members can also be granted permissions to create teams
```

File: content/docs/intro/console/extensions/ci-cd-integration-assistant.md
Comment (davidwrede) [inline content/docs/intro/console/extensions/ci-cd-integration-assistant.md:65]:
```suggestion
**Configure VCS** step are complete, select the **Next** button. The wizard will confirm
```

**Review by davidwrede — COMMENTED** [review]
Looks good! Just some minor nits and copyedit suggestions.

**Comment by davidwrede** [comment]
> @davidwrede i noticed we used a mix of `admin` and `administrator` in the console docs, would you mind if i standardized to `admin`, that's what we use in the console itself?

Yeah, totally. Let's standardize it to `admin` to be consistent.

File: content/docs/intro/console/collaboration/organization-roles.md
Comment (susanev) [reply content/docs/intro/console/collaboration/organization-roles.md:58]: ugh, thx, sry.

File: content/docs/intro/console/collaboration/teams.md
Comment (susanev) [reply content/docs/intro/console/collaboration/teams.md:23]: yes

**Review by davidwrede — APPROVED** [review]

**Comment by susanev** [comment]
im going to be out tomorrow, but folks can merge this after https://github.com/pulumi/pulumi-service/pull/6410 and https://github.com/pulumi/pulumi-service/pull/6395 make it production.

File: content/docs/intro/console/accounts-and-organizations/accounts.md
Comment (cnunciato) [inline content/docs/intro/console/accounts-and-organizations/accounts.md:34]:
```suggestion
To edit your profile information, navigate to **Profile Settings** by selecting
your avatar, then **Settings**. The **Profile** tab is displayed by default.
```

File: content/docs/intro/console/accounts-and-organizations/organizations.md
Comment (cnunciato) [inline content/docs/intro/console/accounts-and-organizations/organizations.md:70]: Little thing, but since this PR is about changing Console terminology and these words instead refer to GitHub, GitLab and Atlassian "admin-type things", we should probably make sure we're using the right terminology for each provider, just to avoid any confusion.

File: content/docs/intro/console/accounts-and-organizations/organizations.md
Comment (cnunciato) [inline content/docs/intro/console/accounts-and-organizations/organizations.md:131]:
```suggestion
1. Select the organization drop-down list in the upper left corner of the Console screen
```

File: content/docs/intro/console/accounts-and-organizations/organizations.md
Comment (cnunciato) [inline content/docs/intro/console/accounts-and-organizations/organizations.md:144]:
```suggestion
| Dashboard | An overview of the organization including recently updated stacks, recent activity, and a resource count graph. |
```

File: content/docs/intro/console/accounts-and-organizations/organizations.md
Comment (cnunciato) [inline content/docs/intro/console/accounts-and-organizations/organizations.md:145]:
```suggestion
| Projects | A searchable list of organization stacks that you can group by project and tag. For more information, see [Project and Stack Management]({{< relref "/docs/intro/console/collaboration/project-and-stack-management" >}}). |
```

File: content/docs/intro/console/accounts-and-organizations/organizations.md
Comment (cnunciato) [inline content/docs/intro/console/accounts-and-organizations/organizations.md:148]: This one's a bit weird in that the others explain what the tab is, whereas this one explains the concept of policies. Seems like it's missing something before it, like

```suggestion
| Policies | Lists of organization policies and policy groups. Policies allow you to set guardrails to enforce best practices and compliance. |
```

File: content/docs/intro/console/accounts-and-organizations/organizations.md
Comment (cnunciato) [inline content/docs/intro/console/accounts-and-organizations/organizations.md:149]:
```suggestion
| Settings | Organization settings including subscription and payment information and history, stack permissions, and links to Pulumi's [continuous delivery guides]({{< relref "/docs/guides/continuous-delivery" >}}). |
```

File: content/docs/intro/console/collaboration/auditing.md
Comment (cnunciato) [inline content/docs/intro/console/collaboration/auditing.md:31]:
```suggestion
only. If you are an organization admin, you can view your organization's audit logs
```

File: content/docs/intro/console/collaboration/auditing.md
Comment (cnunciato) [inline content/docs/intro/console/collaboration/auditing.md:32]:
```suggestion
by selecting your organization from the organization drop down, then selecting the **Settings** tab.
```

"Choosing" the settings tab would be okay here too, just to avoid the "select" repetition.

File: content/docs/intro/console/collaboration/auditing.md
Comment (cnunciato) [inline content/docs/intro/console/collaboration/auditing.md:33]: @davidwrede What's our story on the **bolding** thing? Is this an intentional change? Up to now, we've rarely used bold in prose like this (rarely used it at all in fact), and instead just relied on capitalization, mainly to keep the visual noise level down (and I'm generally a fan of less boldness for this reason -- it's usually just distracting). We should probably clarify one way or another, though, so we can get consistent from here on out. 

(Incidentally my vote would be for no-bold. 😇 🙏 In addition to just looking calmer and being equally clear, it also makes it easier to call attention to the things that really warrant it.)

File: content/docs/intro/console/collaboration/organization-roles.md
Comment (cnunciato) [inline content/docs/intro/console/collaboration/organization-roles.md:59]:
```suggestion
Additionally, organization admins can toggle whether organization members can create stacks, whether
stack admins can delete stacks, and whether stack admins can move stacks to other organizations.
```

File: content/docs/intro/console/collaboration/project-and-stack-management.md
Comment (cnunciato) [inline content/docs/intro/console/collaboration/project-and-stack-management.md:61]:
```suggestion
Selecting "Details" from the Stack update tab, or selecting a specific activity from the login screen, gives you a detailed view of that specific update. The Activity tab provides insights into the operations that were performed on your stack resources during an update. This tab can include three tabs, depending on your setup:
```

File: content/docs/intro/console/collaboration/project-and-stack-management.md
Comment (cnunciato) [inline content/docs/intro/console/collaboration/project-and-stack-management.md:64]: THANK YOU. ❤️ 🍨 ⭐

File: content/docs/intro/console/collaboration/teams.md
Comment (cnunciato) [inline content/docs/intro/console/collaboration/teams.md:21]:
```suggestion
Members of a team can be granted `Team admin` or `Team member` permissions. Team admins can add members to a
```

File: content/docs/intro/console/collaboration/teams.md
Comment (cnunciato) [inline content/docs/intro/console/collaboration/teams.md:23]:
```suggestion
assigned the team member role. To change a team member's role, use the ellipsis menu item at the end of the table row.
```

File: content/docs/intro/console/collaboration/teams.md
Comment (cnunciato) [inline content/docs/intro/console/collaboration/teams.md:29]: Should this maybe be

```suggestion
in the organization's Settings tab. Any member who creates a team will automatically be assigned the team admin role.
```

? It just reads a bit oddly as "in the organization settings."

File: content/docs/intro/console/collaboration/project-and-stack-management.md
Comment (cnunciato) [inline content/docs/intro/console/collaboration/project-and-stack-management.md:71]: Could we maybe update this image also, and bump the indendation a bit to line it up with the list item?

image

**Review by cnunciato — COMMENTED** [review]
Thanks for doing this! We should probably fix a few things up before merging, though.

File: content/docs/intro/console/accounts-and-organizations/organizations.md
Comment (susanev) [reply content/docs/intro/console/accounts-and-organizations/organizations.md:148]: heh yah thx for fixing it, i just pulled that text from the ui

File: content/docs/intro/console/accounts-and-organizations/organizations.md
Comment (susanev) [reply content/docs/intro/console/accounts-and-organizations/organizations.md:70]: on very quick google search they all call them admins

File: content/docs/intro/console/collaboration/project-and-stack-management.md
Comment (susanev) [reply content/docs/intro/console/collaboration/project-and-stack-management.md:71]: ill try and do this tonight, but if not im good merging. as im going to redo all the screenshots right before 4/20 anyway.

File: content/docs/intro/console/collaboration/auditing.md
Comment (davidwrede) [reply content/docs/intro/console/collaboration/auditing.md:33]: Yes, I'd like us to be intentional about bolding important UI elements. This is generally a best practice when documenting steps or items in a UI (e.g. both [Google's developer guide](https://developers.google.com/style/ui-elements#formatting) and the [MSTP style guide](https://docs.microsoft.com/en-us/style-guide/procedures-instructions/formatting-text-in-instructions#in-the-ui-and-general-content) use this convention).

If the bolding itself is too heavy weight, we should probably revisit the font and styling we use on docs to collectively thin the text out a bit.

Comment (susanev) [reply content/docs/intro/console/collaboration/auditing.md:33]: lets wait until we have a new font, then we can figure out the weight to use for ui elements.

File: content/docs/intro/console/collaboration/project-and-stack-management.md
Comment (susanev) [reply content/docs/intro/console/collaboration/project-and-stack-management.md]: @cnunciato i updated 2 more screenshots on this page, i think that whole section needs to be re-wroked though, especially the hierarchy of information. but ive kinda let this pr snowball enough, so wont be taking it on in this pr.

**Pull request merged**
